src/services/microservices-service.js

```diff
@@ -530,6 +535,24 @@ async function updateMicroserviceEndPoint (microserviceUuid, microserviceData, u
   await _updateChangeTracking(true, updatedMicroservice.iofogUuid, transaction)
 }
 
+/**
+ * checks if microservice image is updated
+ * @param {*} microserviceDataUpdateImages
+ * @param {*} catalogImages
+ */
+const checkIfMicorserviceImageIsUpdated = (microserviceDataUpdateImages, catalogImages) => {
```

Comment:
The name is wrong, should be checkIfMicroserviceImageIsEqual , there it looks like the function will return true if they are different

Reply:
updated